maps/tests/Metadata/changeTile.html

```diff
@@ -0,0 +1,24 @@
+<!doctype html>
+<html lang="en">
+    <head>
+        <script src="http://play.workadventure.localhost/iframe_api.js"></script>
```

Comment:
This works only on localhost. We need to find a way to have the `iframe_api.js` relative to the test URL

Reply:
îts not really relative to the test url though 🤔

Reply:
Yeah... it's actually way harder than I first thought....

We could pass the URL of the iframe_api.js in parameter (when opening the iFrame), but that would open up the iFrame to a XSS attack (anyone with the iFrame URL could generate a link that would load any JS in the context if the iFrame, which is probably bad)

Reply:
i mena if you can add a scritp tag you can always execute arbitrary code to your iframe